azure-quantum/azure/quantum/qiskit/qir/translate.py

```diff
@@ -0,0 +1,31 @@
+##
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+##
+
+from types import MethodType
+from typing import Optional
+from qiskit import QuantumCircuit
+
+from azure.quantum.qiskit.qir.quantumcircuit import QirQuantumCircuit
+
+
+def to_qir(
+    circuit: QuantumCircuit,
+    filename: Optional[str] = None,
+    encoding: Optional[str] = None
```

Comment:
Similarly here, it may be good to return a str instead of owning file handling inside the function itself.